catalog-rest-service/src/main/java/org/openmetadata/catalog/CatalogApplication.java

```diff
@@ -267,6 +270,15 @@ private void registerEventPublisher(CatalogApplicationConfig catalogApplicationC
     }
   }
 
+  public void registerKafkaPublisher(CatalogApplicationConfig catalogApplicationConfig) {
+    // register Kafka Event publisher
```

Comment:
lets add UI way of controlling this instead of config. We should model similar to web hook.